fhttp/http_client.go

```diff
@@ -510,29 +511,31 @@ func Fetch(httpOptions *HTTPOptions) (int, []byte) {
 
 // FastClient is a fast, lockfree single purpose http 1.0/1.1 client.
 type FastClient struct {
-	buffer       []byte
-	req          []byte
-	dest         net.Addr
-	socket       net.Conn
-	socketCount  int
-	size         int
-	code         int
-	errorCount   int
-	headerLen    int
-	url          string
-	host         string
-	hostname     string
-	port         string
-	http10       bool // http 1.0, simplest: no Host, forced no keepAlive, no parsing
-	keepAlive    bool
-	parseHeaders bool // don't bother in http/1.0
-	halfClose    bool // allow/do half close when keepAlive is false
-	reqTimeout   time.Duration
-	uuidMarkers  [][]byte
-	logErrors    bool
-	id           int
-	https        bool
-	tlsConfig    *tls.Config
+	buffer             []byte
+	req                []byte
+	dest               net.Addr
+	socket             net.Conn
+	socketCount        int
+	size               int
+	code               int
+	errorCount         int
+	headerLen          int
+	url                string
+	host               string
+	hostname           string
+	port               string
+	http10             bool // http 1.0, simplest: no Host, forced no keepAlive, no parsing
+	keepAlive          bool
+	parseHeaders       bool // don't bother in http/1.0
+	halfClose          bool // allow/do half close when keepAlive is false
+	reqTimeout         time.Duration
+	uuidMarkers        [][]byte
+	logErrors          bool
+	id                 int
+	https              bool
+	tlsConfig          *tls.Config
+	maxConnectionReuse int
```

Comment:
sometimes I add a // above to avoid the full whitespace change
or maybe in this case call it  `maxConnReuse` though that's also still longer than `parseHeaders`
maybe `connReuse` as it's not quite the max but range anyway?

Reply:
fixed